src/glmfit.jl

```diff
@@ -866,4 +862,20 @@ function invloglikhessian(m::GeneralizedLinearModel)
     r = varstruct(m)
     wts = weights(m)
     return inverse(m.pp) * sum(wts) / nobs(m)
-end
\ No newline at end of file
+end
+
+function StatsBase.cooksdistance(m::GeneralizedLinearModel)
+    h = leverage(m)
+    hh = h ./ (1.0 .- h) .^ 2
```

Comment:
To preserve types like `Float32`:
```suggestion
    hh = h ./ (1 .- h) .^ 2
```